Client/Utils/HistoryDeletionUtility.swift

```diff
@@ -102,36 +99,13 @@ class HistoryDeletionUtility: HistoryDeletionProtocol {
 
         switch dateOption {
         case .allTime:
-            switch profile.historyApiConfiguration {
-            case .old:
-                profile.history
-                    .clearHistory()
-                    .uponQueue(.global(qos: .userInteractive)) { result in
-                        completion(result.isSuccess)
-                    }
-            case .new:
-                _ = profile.history
-                    .clearHistory()
-                profile.places.deleteEverythingHistory().uponQueue(.global(qos: .userInteractive)) { result in
-                    completion(result.isSuccess)
-                }
+            profile.places.deleteEverythingHistory().uponQueue(.global(qos: .userInteractive)) { result in
```

Comment:
@OrlaM's question about queue priority applies here too I think. Why .userInteractive and not default?

Reply:
heh that's what the old code was like, and I'm not sure if there is good context 😬 Could be a good candidate for a "investigate all the high priority queue requests" follow up though!

Reply:
Yeah, I might do that related to https://mozilla-hub.atlassian.net/browse/FXIOS-5343